cmd/dockerd/daemon.go

```diff
@@ -83,6 +83,13 @@ func (cli *DaemonCli) start(opts *daemonOptions) (err error) {
 	if cli.Config, err = loadDaemonCliConfig(opts); err != nil {
 		return err
 	}
+
+	if opts.Validate {
```

Comment:
Thanks!

Also wondering if we should move the `stopc` code at line 78 further down;

```go
stopc := make(chan bool)
defer close(stopc)
```

It looks like it's not used until line 190, so moving it closer to that would make it both easier to read, and skips the problem of that defer altogether.